packages/wrangler/src/hyperdrive/shared.ts

```diff
@@ -0,0 +1,12 @@
+export function capitalizeScheme(scheme: string | undefined) {
+	let ret = "";
+
+	if (scheme === "mysql") {
+		ret = "MySQL";
+	}
+	if (scheme === "postgres" || scheme === "postgresql") {
+		ret = "PostgreSQL";
+	}
+
+	return ret;
```

Comment:
NIT this could simply be:

```suggestion
	switch (scheme) {
	  case "mysql":
		 return "MySQL";
	  case "postgres":
	  case "postgresql":
		  return "PostgreSQL";
		default:
		  return "";
	}
```